.github/workflows/test.yml

```diff
@@ -0,0 +1,53 @@
+---
+name: Tests
+permissions:
+  contents: read
+on:
+  pull_request:
+    paths-ignore:
+      - "*.md"
+      - "assets/**"
+  push:
+    paths-ignore:
+      - "*.md"
+      - "assets/**"
+    branches:
+      - main
+  workflow_dispatch:
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  ubuntu:
+    name: Test
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Install tmux
+        run: sudo apt-get update && sudo apt-get install -y tmux
```

Comment:
We should pin the version of tmux here so that the version of tmux doesn't implicitly change between runs.

Reply:
Is `apt-get install tmux=3.2a-4ubuntu0.2` fixed enough?

Reply:
Does a later version such as https://github.com/tmux/tmux/releases/tag/3.4 exist in apt? I just wanted to make sure it was pinned to at least one of the versions in the releases

Reply:
`ubuntu-latest` action runner is [22.04/jammy](https://packages.ubuntu.com/jammy/tmux) for which 3.2a the latest is. Compile from source?

Reply:
Ah no worries, 3.2a is fine :+1: 
Don't think its worth it for a simple test